conda_forge_tick/migrators/core.py

```diff
@@ -721,7 +721,18 @@ def commit_message(self, feedstock_ctx: FeedstockContext) -> str:
 
     def pr_title(self, feedstock_ctx: FeedstockContext) -> str:
         assert isinstance(feedstock_ctx.attrs["new_version"], str)
-        return feedstock_ctx.package_name + " v" + feedstock_ctx.attrs["new_version"]
+        # TODO remove this later when we want to automerge everything
+        if feedstock_name.feedstock_name == 'cf-autotick-bot-test-package':
```

Comment:
This is correct, we don't have `'feedstock'` in the `feedstock_name` since everything would have that name.

Reply:
We also have access to the `conda-forge.yaml` if you want to use that to check if automerge is enabled.

Reply:
Interesting. I think it makes sense to leave that check in the action, not here. That way if someone changes the automerge setting on master, old PRs will still have the slug and so can be merged.

Reply:
But we can work this out later. I want to simply turn things on for the testing feedstock for now.